modules/javafx.graphics/src/test/java/test/com/sun/javafx/iio/ImageStorageTest.java

```diff
@@ -84,4 +85,11 @@ public void testCorruptFirstFrame() throws ImageStorageException  {
         String path = getResourcePath("gif/animation/testBad.gif");
         ImageStorage.loadAll(path, null, 0, 0, false, 1.0f, false);
     }
+
+    @Test
+    public void testLoadAllFramesAnimatedGIF() throws ImageStorageException {
+        String path = "https://upload.wikimedia.org/wikipedia/commons/2/2c/Rotating_earth_%28large%29.gif";
```

Comment:
Except for specific network tests we don't want to load anything from the internet during a run. For one thing this will fail behind a firewall (without setting a proxy, which we don't want to require). For another, the content could change out from under us.

The best solution would be to generate a synthetic image with properties that will cause it to fail (we can't use a preexisting image due to copyright issues).

Reply:
Okay, I've replaced the test with an encoded image of a single frame. It fails with IAOOB without the fix, passes with it.